src/index.html

```diff
@@ -1,13 +1,63 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no,   initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.  0"
+    >
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,900&display=swap" rel="stylesheet"
+    >
+
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product" data-qa="card">
+      <img 
+        class="product__foto" 
+        src="./images/imac.jpeg" 
+        alt="product__foto"
+        width="160"
+        height="134"
+      >
+    
+      <div class="product__info">
+        <div class="product__name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+        <div class="product__code">Product code: 195434</div>
+        <div class="product__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <span class="reviews">Reviews: 5</span>
+        </div>
+
+        <div class="product__price">
+          <span class="product__price-name">Price:</span>
+          <span class="product__price-value">$2,199</span>
+        </div>
+        <a 
+          href="/"
+          data-qa="hover"
+        >
+          <button class="buy-btn">Buy</button>
```

Comment:
```suggestion
          <button class="card__button">Buy</button>
```
It is not a separate BEM block, just an element of card